src/adcs_handler.c

```diff
@@ -65,6 +66,91 @@ ADCS_returnState adcs_telemetry(uint8_t TM_ID, uint8_t* reply,
   return ack;
 }
 
+/**
+ * @brief
+ * 		append two bytes of a int16_t
+ * @param b1 LSB, b2 MSB
+ * @return
+ * 		the appended int16_t
+ */
+int16_t uint82int16(uint8_t b1, uint8_t b2) {
+  int16_t b;
+  if (b2 >> 7) {  // negative
+    b = -1 * ((~((b2 << 8) | b1)) + 1);
+  } else
+    b = (b2 << 8) | b1;  // positive
+  return b;
+}
+
+/**
+ * @brief
+ * 		append four bytes of a int32_t
+ * @param address
+ * 		the address of the 4 bytes
+ * @return
+ * 		the appended int32_t
+ */
+int32_t uint82int32(uint8_t* address) {
+  int32_t b;
+  if (*(address + 3) >> 7) {  // negative
+    b = -1 * ((~((*(address + 3) << 24) | (*(address + 2) << 16) |
+                 (*(address + 1) << 8) | *address)) +
+              1);
+  } else
+    b = (*(address + 3) << 24) | (*(address + 2) << 16) |
+        (*(address + 1) << 8) | *address;  // positive
+  return b;
+}
+
+/**
+ * @brief
+ * 		append two bytes of a uint16_t to a float
+ * @param b1 LSB, b2 MSB
+ * @return
+ * 		the appended float
+ */
+uint16_t uint82uint16(uint8_t b1, uint8_t b2) {  //* improve
```

Comment:
To Do: We should put these functions into a new file so we can use them in `test`, too. A lot of bitwise operations in this file can be replaced with this function (probably with a better name!).